style.css

```diff
@@ -1,340 +1,1338 @@
+    margin-bottom: 15px;
+    padding: 24px;
+    background-color: rgba(29, 28, 40, 1);
+}
+
+.faq__question {
+    font-family: var(--font-secondary);
+    font-size: 24px;
+    line-height: 32px;
+    text-underline-position: from-font;
+    text-decoration-skip-ink: none;
+}
+
+.faq__answer {
+    margin-top: 20px;
+    padding-top: 20px;
+    border-top: 1px solid rgba(191, 197, 238, 0.12);
+}
+
+details>summary {
+    list-style: none;
+}
+
+summary::after {
+    position: absolute;
+    content: url('images/plus.svg');
+    right: 24px;
+}
+
+details[open] summary:after {
+    position: absolute;
+    content: url('images/minus.svg');
+    right: 24px;
+}
+
+/* Footer */
+
+.footer {
+    background-color: #1A1925;
+}
+
+.footer__main {
+    display: flex;
+    justify-content: space-between;
+    max-width: 1630px;
+    padding: 60px 15px 40px 15px;
+    margin: 0 auto;
+    border-bottom: 1px solid #BFC5EE1F;
+}
+
+.footer .logo__text {
+    color: #0265DC;
+}
+
+.footer-nav {
+    display: flex;
+}
+
+.footer-nav__block {
+    margin-left: 20px;
+}
+
+.footer-nav__title {
+    display: block;
+    padding-bottom: 10px;
+    font-size: 20px;
+    font-weight: 500;
+    line-height: 28px;
+}
+
+.footer-nav__item {
+    padding: 2px 30px 2px 0;
+}
+
+.footer__description {
+    max-width: 320px;
+    font-size: 14px;
+    line-height: 20px;
+}
+
+.footer__secondary {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    max-width: 1630px;
+    margin: 0 auto;
+    padding: 10px 15px;
+}
+
+.footer__copyright {
+    font-size: 14px;
+    line-height: 20px;
+}
+
+.footer__copyright span {
+    text-transform: uppercase;
+}
+
+.footer__social img {
+    vertical-align: top;
+}
+
+.catalog,
+.blog,
+.about,
+.faq {
+    margin-bottom: 80px;
+}
+
+
+
+
+
+
+/*Blog page */
+
+.blog__main {
+    display: flex;
+    max-width: 1280px;
+    margin: 0 auto;
+}
+
+.blog-intro {
+    position: relative;
+    display: flex;
+    justify-content: center;
+    height: 375px;
+    margin: 0 auto;
+    background-color: #14131C;
+}
+
+.blog-intro::before {
+    width: 100%;
+    height: 100%;
+    content: '';
+    position: absolute;
+    background-image: url('images/bg-3.png');
+    background-size: cover;
+    background-position: center;
+    background-repeat: no-repeat;
+    z-index: 1;
+    opacity: 20%;
+}
+
+.blog-intro::after {
+    width: 100%;
+    height: 100%;
+    content: '';
+    position: absolute;
+    background-image: url('images/bg-4.png');
+    background-size: contain;
+    background-position: bottom;
+    background-repeat: no-repeat;
+    z-index: 2;
+}
+
+.blog-intro__wrapper {
+    position: absolute;
+    padding-top: 65px;
+    z-index: 3;
+}
+
+.blog-intro__wrapper>.intro__title {
+    text-transform: none;
+    text-align: center;
+}
+
+.breadcrumbs {
+    display: flex;
+    justify-content: center;
+}
+
+.breadcrumbs__item {
+    position: relative;
+    padding: 6px 12px;
```

Comment:
Move color: rgba(231, 238, 247, 0.66); here (all text inside li would be with this color, except links)

Reply:
done